show-land/sass/layout/_super-fast.scss

```diff
@@ -23,7 +22,8 @@
     align-items: baseline;
     gap: 4px;
     background-image: linear-gradient(to right, $colorSecondary, $colorPrimary);
-    padding: nth($spaces, 1) nth($spaces, 4) nth($spaces, 1) nth($spaces, 4);
+    padding: list.nth($spaces, 1) list.nth($spaces, 4) list.nth($spaces, 1)
+      list.nth($spaces, 4);
```

Comment:
you can only write this code :
`padding: list.nth($spaces, 1) list.nth($spaces, 4)`